Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM node:14
+
+LABEL maintainer="Ryan Blunden <ryan.blunden@ext.doppler.com>"
+
+# DOPPLER_TOKEN must be supplied as a build-arg
+ARG DOPPLER_TOKEN
+ENV DOPPLER_TOKEN ${DOPPLER_TOKEN}
+
+# Install the Doppler CLI
+RUN curl -Ls https://cli.doppler.com/install.sh | sh
+
+# Execute `doppler run` so it creates a secrets fallback file
+RUN doppler run -- echo "Create secrets fallback file"
+
+WORKDIR /usr/src/app
+COPY ["package.json", "package-lock.json*", "src", "./"]
+RUN npm install --production --silent && mv node_modules ../
```

Comment:
I'm confused by this `mv`- won't it make the `node_modules` inaccessible?

Reply:
This was generated from Microsoft's Node Docker generation template. I can only presume Node traverses up the directory tree looking for a `node_modules` directory. It's a cool trick, as it means you can easily do local development by mounting in the current dir without having to sync the `node_modules` directory between the host and the container.

Reply:
That is a cool trick. It's giving me some ideas for speeding up our server's local docker builds.